webapp/public/index.html

```diff
@@ -45,6 +45,7 @@
     <script type="text/javascript" src="/blb/blockly/blockly_compressed.js"></script>
     <script type="text/javascript" src="/blb/blockly/blocks_compressed.js"></script>
     <script type="text/javascript" src="/blb/blockly/msg/js/en.js"></script>
+    <script type="text/javascript" src="/blb/smoothie/smoothie.js"></script>
```

Comment:
any chance to minify?

Reply:
Yes, minify.

Reply:
Use smoothie_compressed instead

Reply:
Post-beta, per discussion with Abhijith

Reply:
See https://github.com/Microsoft/pxt/issues/3124